src/main/java/expiryeliminator/commands/ListRecipeCommand.java

```diff
@@ -0,0 +1,25 @@
+package expiryeliminator.commands;
+
+
+import expiryeliminator.data.IngredientList;
+import expiryeliminator.data.RecipeList;
+import expiryeliminator.data.exception.DuplicateDataException;
+
+/**
+ * Lists all recipes, together with its associated ingredients.
+ */
+public class ListRecipeCommand extends Command {
+
+    /** Unique word associated with the command. */
+    public static final String COMMAND_WORD = "list recipes";
+
+    private static final String SHOW_WHOLE_RECIPE_LIST = "Here are the recipes in your recipe list:\n" + "\n%1$s\n"
```

Comment:
Perhaps rename this as `MESSAGE_RECIPE_LIST`? This is to make it standardised because we usually start messages with `MESSAGE_`